src/libraries/Microsoft.Extensions.Primitives/src/ThrowHelper.cs

```diff
@@ -53,7 +53,7 @@ internal static ArgumentException GetArgumentException(ExceptionResource resourc
 
         private static string GetResourceText(ExceptionResource resource)
         {
-            return Resources.ResourceManager.GetString(GetResourceName(resource), Resources.Culture);
+            return SR.GetResourceString(GetResourceName(resource));
```

Comment:
Can you spot check the behavior here to make sure this is working correctly?

Reply:
tested locally, works correctly